spec/models/user_spec.rb

```diff
@@ -79,13 +79,13 @@
     describe "#project_matches" do
       it "returns a project's skills and interests that match the user's" do
         expect(user.project_matches(most_relevant_project).include?(
-          {"interest" => "film"}))
+          "interest" => "film"))
```

Comment:
Indent the first parameter one step more than `user.project_matches(most_relevant_project).include?(`.